libsolidity/ast/Types.h

```diff
@@ -738,6 +738,7 @@ class EnumType: public Type
 	EnumDefinition const& enumDefinition() const { return m_enum; }
 	/// @returns the value that the string has in the Enum
 	unsigned int memberValue(ASTString const& _member) const;
+	size_t numberOfMembers() const;
```

Comment:
At this location, `class EnumDefinition` is only forward-declared, so I cannot look at its contents.